test/distributed/org/apache/cassandra/distributed/test/OptimiseStreamsRepairTest.java

```diff
@@ -107,7 +106,7 @@ public void testBasic() throws Exception
     {
         public static void install(ClassLoader cl, int id)
         {
-            new ByteBuddy().rebase(RepairJob.class)
+            new ByteBuddy().rebase(AbstractRepairJob.class)
```

Comment:
should this target `CassandraRepairJob`?

Reply:
Oh yikes and the test still passes. I'll fix that as well and make sure the injected version is getting called.